Project.toml

```
@@ -19,5 +19,6 @@ MPI = "0.20"
 MPIPreferences = "0.1"
 Preferences = "1"
 Reexport = "0.2, 1.0"
+TOML = "1"
```

```suggestion
TOML = "1.6"
```
It should have the same version number as the Julia entry here - see https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958

---

Looks like the announcement messed something up. Sorry for the noise! I think having `"1"` here should be fine 👍